rkovalenko/ReverseLinkedList.cs

```diff
@@ -0,0 +1,37 @@
+/**
+ * Definition for singly-linked list.
+ * public class ListNode {
+ *     public int val;
+ *     public ListNode next;
+ *     public ListNode(int val=0, ListNode next=null) {
+ *         this.val = val;
+ *         this.next = next;
+ *     }
+ * }
+ */
+public class Solution {
```

Comment:
Interesting take on the problem - another way to solve!

Reply:
Yep, without links manipulation required two passes on a list and some additional memory. Good example.